OneCommunityGlobal/HighestGoodNetworkApp
# Urgent - Ayush - Designed the star and the dot according to the logic

# Description

First Task :Make the dot by people’s names show if they’ve done significant additional hours.

Dot by person's name in the Leaderboard: I'd like this to show:
i. As lighter green if a person has done 25-49.9% more than their committed hours 
ii. As fuchsia if a person has done 50%-74.9% more than their committed hours
iii. As purple if a person has done 75%-100+% more than their committed hours

Second Task : Make the star by people’s names show if they’ve done significant additional hours.
(Reports -> Weekly Summary Reports)


Show the star only if the hours logged is greater than the committed hours. 
Include the percentage of time they have done extra over there alloted time.

i. As lighter green if a person has done 25-49.9% more than their committed hours 
ii. As fuchsia if a person has done 50%-74.9% more than their committed hours
iii. As purple if a person has done 75%-100+% more than their committed hours

## How to test:
check into current branch
do `npm install` and `...` to run this PR locally
log as admin user

See whether the dot and the star colors are coming as the above logic


## Separate PR

1 - For the stars, reminder they should show on the Leaderboard only for Owner, Admin, Manager, Mentor and Core Team classes. Currently all classes except Volunteer, but please specifically code it for those classes so that new classes won’t get this ability by default.
2- As a separate PR, we’ll need to add “View Stars for Exceeding Min Hrs by 25/50/75%” to the Permissions Management Page too. Then test that it can be added/removed from each class and added to a new made up class and/or an individual. 

## Screenshots or videos of changes:

## Note:
Include the information the reviewers need to know.
